Deploy issues

Testing in Chrome stable after a Netlify deploy a few things I noticed:

* UX: We might want to only show recent games when there are more than one with data available
* `VITE_API_BASE_URL` It was unclear to me what this should be (from the logs below, I did try to check the README and env)
* Over chat it was mentioned this being Netlify vs Vercel functions may play a role. We might need to migrate current to Netlify functions: https://docs.netlify.com/functions/overview/

<img width="1501" alt="Image" src="https://github.com/user-attachments/assets/542df7af-2d35-4a8e-b5bd-3a2d066f65da" />

```js
build.js:2667 Vercel API request failed: TypeError: Cannot read properties of undefined (reading 'VITE_API_BASE_URL')
    at GameManager.generateGameSummaryGemini (build.js:2624:42)
    at GameManager.generateGameSummary (build.js:2755:36)

build.js:2760 Error generating game summary: TypeError: Cannot read properties of undefined (reading 'VITE_API_BASE_URL')
    at GameManager.generateGameSummaryGemini (build.js:2624:42)
    at GameManager.generateGameSummary (build.js:2755:36)
```

--- a/js/build.js
+++ b/js/build.js
@@ -2849,7 +2849,9 @@ class GameManager {
     this.cancelStarter();
     // Enable touch controls when game starts
     input.enableTouchControls();
-    this.interface.other.recentScores.classList.remove("hidden");
+    if (score.recent_scores.length > 0) {
+      this.interface.other.recentScores.classList.remove("hidden");
+    }
     clock.getDelta();
     this.render();
     this.loop();

--- a/netlify.toml
+++ b/netlify.toml
@@ -1,0 +1,12 @@
+[build]
+  command = "npm run build"
+  publish = "dist"
+  functions = "api"
+
+[functions]
+  directory = "api"
+
+[[redirects]]
+  from = "/api/*"
+  to = "/.netlify/functions/:splat"
+  status = 200 

